support_bundle/get_support_bundle.sh

```diff
@@ -480,6 +480,14 @@ main() {
     echo "Fetching the sysdigcloud-config ConfigMap"
     kubectl ${KUBE_OPTS} get configmap sysdigcloud-config -o yaml | grep -v password | grep -v apiVersion > ${LOG_DIR}/config.yaml || true
 
+    # Collect the sysdigcloud-values secret, and write to the log directory if the backend version is 6.5 or higher
+    major=$(echo $version | awk -F. '{ print $1 }')
+    minor=$(echo $version | awk -F. '{ print $2 }')
+    if [[ $major -ge 6]] && [[ $minor -ge 5 ]]; then
```

Comment:
this will only work for 6.5 and greater in the 6.x line. It will not work for 7.x or 8.x releases. I think that you want something like
```
if [[ $major -gt 6 ]] || [[ $major -ge 6 ]] && [[ $minor -ge 5 ]];
```